core/src/main/java/ru/tinkoff/kotea/core/KoteaStoreFactory.kt

```diff
@@ -41,7 +41,7 @@ fun <State : Any, Event : Any, UiEvent : Event, Command : Any, News : Any> Kotea
     initialCommands: List<Command> = emptyList(),
     commandsFlowHandlers: List<CommandsFlowHandler<Command, Event>> = emptyList(),
     update: Update<State, Event, Command, News> = Update { _, _ -> Next() },
-    uncaughtExceptionHandler: UncaughtExceptionHandler = UncaughtExceptionHandler { }
+    uncaughtExceptionHandler: UncaughtExceptionHandler = UncaughtExceptionHandler { throw it }
```

Comment:
Hi, it seams like both auto re-throw and ignore  behavior for exceptions could be harmful. Can this library require to **explicitly** force user to put custom logic for handling exceptions instead? This way consumers will at least think twice about how they want to handle errors

Reply:
Ideally, all exceptions should be caught inside their originating `CommandsFlowHandler`s.
So, I think rethrow behaviour as default is OK, because **uncaught** exceptions are usually mean bugs.
Besides, if exception got here, it's already too late to meaningfully recover and we don't need to encourage user to suppress these exceptions.

Reply:
In fact, maybe we don't really need this handler since user can provide a `CoroutineExceptionHandler` as part of the context passed in `launchIn` method. And as all exceptions from command handlers are wrapped in `CommandsFlowHandlerException`, it is easy to distinguish them from exceptions from the `Update`.

Reply:
I removed UncaughtExceptionHendler, right now users can add CoroutineExceptionHandler via storeViaViewModel delegate, it is shown in sample